plugins/modules/ome_session.py

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.3.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ome_session
+short_description: Manage OpenManage Enterprise and OpenManage Enterprise modular sessions
+version_added: "9.3.0"
+description:
+  - This module allows you  to create and delete sessions on OpenManage Enterprise and OpenManage Enterprise Modular.
+options:
+  hostname:
+    description:
+      - IP address or hostname of the OpenManage Enterprise.
+    type: str
+  username:
```

Comment:
Add Env variables for Creds, update for idrac session as well.

Reply:
done